lib/CLI.rb

```diff
@@ -0,0 +1,34 @@
+class CLI
+  def initialize
```

Comment:
Typically this method would be called something like `call` and your bin would call on `CLI.new.call`.